e2etest/src/test/java/fk/prof/utils/TestBackendVerticle.java

```diff
@@ -0,0 +1,38 @@
+package fk.prof.utils;
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Future;
+import org.apache.commons.lang3.mutable.Mutable;
+import org.apache.commons.lang3.mutable.MutableInt;
+import org.apache.commons.lang3.mutable.MutableObject;
+
+/**
+ * @understands
+ */
+public class TestBackendVerticle extends AbstractVerticle {
```

Comment:
Ensure this class (removed from #2) does not get added back when you merge #3 on top of #2.